examples/resources/pingfederate_captcha_providers/import.sh

```diff
@@ -0,0 +1 @@
+terraform import pingfederate_captcha_providers.captchaProvider captchaProvidersExample
```

Comment:
The word after the `.` should match the name of the resource in the example. And the last word should be the example id, so something like `captchaProviderId`